src/dynamics/joint/generic_joint.rs

```diff
@@ -206,6 +206,8 @@ pub struct GenericJoint {
     ///
     /// Note that the mostor must also be explicitly enabled by the `motors` bitmask.
     pub motors: [JointMotor; SPATIAL_DIM],
+    /// The flag indicating that motor settings were changed from outside
+    pub motor_changed: bool,
```

Comment:
While I appreciate the "low number of lines" this PR ends up, this would add state to an otherwise "configuration" or "data oriented" struct, I'm not sure this is the right place to add that.